common/utils/src/main/resources/error/error-classes.json

```
@@ -1,4 +1,32 @@
 {
+  "AGGREGATE_FILTER_EXPRESSION_ERROR" : {
+    "message" : [
+      "FILTER expression in aggregate function has errors:"
```

When an user query is big enough, it is hard to identify which filter the error belongs to. Let's help to user, and point out the filter expression:
```suggestion
      "The FILTER expression <filterExpr> in an aggregate function is invalid."
```

---

Done as suggested.